go/apps/api/openapi/openapi-generated.yaml

```
@@ -2203,12 +2655,58 @@ components:
             description: Array of API keys with complete configuration and metadata.
+            example:
+                - keyId: key_1234567890abcdef
+                  start: sk_test_abc123
+                  enabled: true
+                  name: Production API Key
+                  createdAt: 1701425400000
+                  updatedAt: 1701425400000
+                  expires: 1735689600000
+                  externalId: user_12345
+                  meta:
+                    plan: premium
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 22 but found 20

(indentation)